src/main/resources/feedbackQuestionRankOptionsEditFormTemplate.html

```diff
@@ -29,16 +29,17 @@
       <div class="col-sm-9 col-xs-12">
         <div class="checkbox" data-toggle="tooltip" data-placement="top" data-container="body" title="Ticking this will ensure respondent ranks at least the mentioned number of options.">
           <label class="bold-label">
-            <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSCHECKBOX}"
-                id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSCHECKBOX}-${questionNumber}" ${isMinOptionsToBeRankedEnabled}>
+            <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_CHECKBOX}"
+                id="${Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_CHECKBOX}-${questionNumber}"
+                   data-linked-input-id="${Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_TO_BE_RANKED}-${questionNumber}" ${isMinOptionsToBeRankedEnabled}>
             Minimum number of options a respondent must rank
           </label>
         </div>
       </div>
       <div class="col-sm-3 col-xs-12">
         <div class="pull-right">
-          <input type="number" class="form-control" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSTOBERANKED}"
-            id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKMINOPTIONSTOBERANKED}-${questionNumber}" min="1" step="any" value="${minOptionsToBeRanked}"/>
+          <input type="number" class="form-control" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_TO_BE_RANKED}"
+            id="${Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_OPTIONS_TO_BE_RANKED}-${questionNumber}" min="1" step="any" placeholder="e.g. 1" value="${minOptionsToBeRanked}" required>
```

Comment:
@Crphang I guess this means we are not having any trouble adding `required` directly with this workaround? (Sorry, I haven't tested this yet.)

Reply:
@sukanta-27 yes. It's fine already.